configs/config.local.yml

```
@@ -0,0 +1,144 @@
+
+# External APIs
+external_apis:
+  coingecko:
+    base_url: https://api.coingecko.com/api/v3
+    timeout: 10s
+    rate_limit: 50
+  coinmarketcap:
+    base_url: https://pro-api.coinmarketcap.com/v1
+    api_key: ""
+    timeout: 10s
+    rate_limit: 333
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)